### PR TITLE
[BLOG] Add the-new-spyder-editor-documentation-under-the-spotlights.md

### PR DESCRIPTION
**Pull request by hyounes4560**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by noatamir** [comment]
@hyounes4560 the blog preview works now. For some reason, the hero image is showing up as white. I was looking at your documentation link and saw you have plenty of great screenshots in there. Would you like to maybe use one or two of those for the blog? You just need to crop them for the hero and feature image sizes: hero 1440x696 pixels, feature 391x174 pixels. If not that ok, please let us know, and we can investigate, why the current hero svg isn't displaying correctly.

File: apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+title: 'The new Spyder Editor documentation under the spotlights!'
+published: October 31, 2022
+author: hanan-younes
+description: 'In this blogpost, I share my experience as a Google Season of Docs 2022 technical writer working on updating the Editor user documentation.'
+category: [Community, OSS Experience]
+featuredImage:
+  src: /posts/the-new-spyder-editor-documentation-under-the-spotlights/blog_feature_var3.svg
+  alt: 'An illustration of a brown and a white hand coming towards each other to pass a business card with the logo of Quansight Labs'
+hero:
+  imageSrc: /posts/what-do-we-need-to-write/blog_hero_var3.svg
```
Comment (noatamir) [inline apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md:11]: I think the path here is mis-specified 😉  If you update the folder name the hero appear.

File: apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+As an open-source software user, I have always felt frustrated when using software
+with out-of-date and unreliable user documentation. The lack of proper
+documentation, especially for newcomers, is a common yet critical issue in many open-source projects.
+Most of these projects offer a lot of information scattered everywhere and lack
+organization, which makes contributing to these projects terrifying, especially
+for people with no prior experience! That’s why I had a goal in mind to make
+contributing and using open-source software less intimidating and more
+convenient, which led me to the Google Season of Docs 21 and 22 programs.
+
+During the past 6 months, I, my mentor for this program "Stephannie Jiménez",
```
Comment (steff456) [inline apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md:31]: I think we can remove the quotes here.

```suggestion
During the past 6 months, I, my mentor for this program Stephannie Jiménez,
```

**Review by steff456 — CHANGES_REQUESTED** [review]
I left a minor comment, but the blog post looks great!

File: apps/labs/posts/nebari-typer-cli.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
+title: 'Developing a Typer CLI for Nebari'
+published: November 7, 2022
```
Comment (noatamir) [inline apps/labs/posts/nebari-typer-cli.md:3]:
```suggestion
published: November 15, 2022
```

Comment (noatamir) [reply apps/labs/posts/nebari-typer-cli.md:3]: update published date

File: apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+title: 'The new Spyder Editor documentation under the spotlights!'
+published: October 31, 2022
```
Comment (noatamir) [inline apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md:3]:
```suggestion
published: Nevomeber 15, 2022
```

**Review by noatamir — APPROVED** [review]
LGTM

**Pull request merged**